tests/v2/validation/auth/userretention/userretention_settings_test.go

```diff
@@ -0,0 +1,225 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package userretention
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rancher/machine/libmachine/log"
```

Comment:
is there a reason why we are using this instead of "github.com/sirupsen/logrus"?